frontend/pages/queries/edit/components/EditQueryForm/EditQueryForm.tsx

```diff
@@ -836,6 +897,23 @@ const EditQueryForm = ({
               >
                 Observers can run
               </Checkbox>
+              {isPremiumTier && (
+                <TargetLabelSelector
+                  selectedTargetType={selectedTargetType}
+                  selectedCustomTarget=""
+                  selectedLabels={selectedLabels}
+                  className={`${baseClass}__target`}
+                  onSelectTargetType={setSelectedTargetType}
+                  onSelectLabel={onSelectLabel}
+                  labels={labels || []}
+                  customHelpText={
+                    <span>
```

Comment:
There's a slight style discrepancy between here and figma. Adding the `"form-field__help-text"`
class puts the element under [these global form field style
rules](https://github.com/fleetdm/fleet/blob/ecbf8f36fd61dca8623e625d45e031a7c1d9a8a0/frontend/styles/global/_global.scss#L138-L162),
which is handy

Reply:
Done!